ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClient.java

```diff
@@ -78,6 +78,14 @@ void createTopic(
    */
   boolean isTopicExists(String topic);
 
+  /**
```

Comment:
Could you write one or two lines about what it checks to see if the topic exists? A quick look at this call made me think it only uses the topic name, but seems it compares all 3 parameters, right?

Reply:
After talking to @big-andy-coates I removed this functionality and instead I won't bother checking if the topic exists at all. This really simplifies the code and makes it so that the physical plan building won't fail if the topic doesn't exist, rather only the query will.